src/org/parosproxy/paros/view/FindDialog.java

```diff
@@ -48,6 +53,9 @@
 public class FindDialog extends AbstractDialog {
 
 	private static final long serialVersionUID = -3223449799557586758L;
+	
+	@SuppressWarnings("unchecked")
+	private static Map<Object, FindDialog> parentsList = new ReferenceMap(ReferenceMap.WEAK, ReferenceMap.HARD);

```

Comment:
This should be renamed, the name implies a list but it's a map (same for the getter).